production.ini.in

```diff
@@ -21,7 +21,8 @@ pyramid.includes = pyramid_tm
 sqlalchemy.are.url = postgresql://${dbhost}:${dbport}/are_${dbstaging}
 sqlalchemy.bafu.url = postgresql://${dbhost}:${dbport}/bafu_${dbstaging}
 sqlalchemy.bak.url = postgresql://${dbhost}:${dbport}/bak_${dbstaging}
-sqlalchemy.bod.url = postgresql://${dbhost}:${dbport}/bod_${dbstaging}
+# TODO: once we want this in production, change back and change the links in bod master layer_js table.
+sqlalchemy.bod.url = postgresql://${dbhost}:${dbport}/bod_ltkum_master
```

Comment:
do not forget to switch back to `bod_master` before going live

Reply:
I'll do my best